backend/src/models/base/base.py

```diff
@@ -0,0 +1,17 @@
+import re
+from uuid import UUID, uuid4
+
+from sqlalchemy.ext.declarative import as_declarative
+from sqlalchemy.orm import Mapped, declared_attr, mapped_column
+
+
+@as_declarative()
+class BaseModel:
+    @classmethod
+    @declared_attr
+    def __tablename__(cls):
+        return re.sub(r"(?<!^)(?=[A-Z])", "_", cls.__name__).lower() + "s"
+
+    id: Mapped[UUID] = mapped_column(
```

Comment:
Don't think that using UUID is required here. It's nice when we have distributed system, but since we will have only one database... Do you think that it makes sense?

Reply:
At least we won't have problems with sequence of AutoIncrement field + it's safer to use 100% unique IDs

Reply:
in myщцт opinion it's better to use UUID as the primary key because a more unique key and in the future you will not face difficulties if your application starts to grow